conversion_proxy/src/lib.rs

```diff
@@ -137,39 +158,47 @@ impl ConversionProxy {
     }
 
     #[init]
-    pub fn new(oracle_account_id: AccountId, provider_account_id: AccountId) -> Self {
+    pub fn new(feed_parser: AccountId, feed_address: Vec<u8>) -> Self {
         let owner_id = env::signer_account_id();
+        let feed_payer = env::signer_account_pk();
```

Comment:
Am I reading this right? Are you storing the owner's Private Key as the public `feed_payer` attribute?

Reply:
@MantisClone Public key 👍 What Switchboard protocol expects.